src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -89,14 +89,6 @@ async def handle_tunable_change(self, tunable):
             await self.middleware.call('etc.generate', 'udev')
             await run(['udevadm', 'control', '-R'])
 
-    @accepts()
```

Comment:
How did you determine this wasn't used? https://github.com/truenas/webui/pull/11376. Seems you looked at old sources.